static/app/gettingStartedDocs/javascript/react.tsx

```diff
@@ -69,6 +71,27 @@ export const steps = ({
       },
     ],
   },
+  {
+    language: 'html',
+    type: StepType.CONFIGURE_SOURCE_MAPS,
+    configurations: [
+      {
+        description: t(
+          'The easiest way to configure upload Source Maps with Webpack is by using the Sentry Wizard.'
```

Comment:
We should not mention webpack here as we don't know what they are using (wizard will let them choose this).